test/osml/model_runner/testing/mr_imagery.test.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright 2024 Amazon.com, Inc. or its affiliates.
+ */
+
+describe("MRSync", () => {
+  beforeEach(() => {});
+
+  it("sets removal policy based on prodLike flag", () => {
+    // Get the removal policy
+  });
+
+  it("creates image bucket", () => {
+    // expect(mrImagery.imageBucket).toBeDefined();
```

Comment:
Do we want this commented out?

Reply:
I was working on it, it will be uncommented soon once I figure out how to pass a custom assets or mock it.